src/components/side-panel/map-manager.js

```diff
@@ -209,7 +213,7 @@ const LayerGroupSelector = ({layers, editableLayers, onChange, topLayers}) => (
             <PanelHeaderAction
               id={`${slug}-top`}
               tooltip="Move to top of data layers"
-              disabled={!layers[slug]}
+              disabled={!layers[slug] || slug === '3d building'}
```

Comment:
This is for the "move to the top" icon on the right (see img below). I feel like it doesn't make sense to move the 3d building layer to the top, and the logic to move this layer may be hard to implement because it's a deckgl layer, so I always set this button to be disabled.  
![image](https://user-images.githubusercontent.com/19985016/47938036-f5cfc200-de9f-11e8-8c26-7158f9b4cff0.png)